SplitWise/src/com/workattech/sw/service/SplitWiseService.java

```diff
@@ -0,0 +1,106 @@
+package com.workattech.sw.service;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.workattech.sw.models.Expense;
+import com.workattech.sw.models.Money;
+import com.workattech.sw.models.User;
+
+public class SplitWiseService {
+	ArrayList<User> userList=new ArrayList<User>();
+	CalculateExpense calculateExpense=new CalculateExpense();
+	public Map<String,ArrayList<Money>> listOfUserWhoOwesToUser=new HashMap<>();
+	public Map<String,ArrayList<Money>> getlistOfUserWhoOwesToUser()
+	{
+		
+		return this.listOfUserWhoOwesToUser;
+	}
+	public void setUserList(ArrayList<User> userList)
+	{
+		this.userList=userList;
+	}
+	public void distributeMoneyIfEqual(ArrayList<String> borrowers,String lender,int amount)
+	{
+		
+		double amountToBeDistributed=calculateExpense.CalculateMoneyEqual(amount,borrowers.size()+1);
+		for(int i=0;i<borrowers.size();i++) {
+		addMoneyToLender(borrowers.get(i),lender,amountToBeDistributed);
+		subtractMoneyFromBorrowers(borrowers.get(i),lender,amountToBeDistributed);
+		}
+	}
+	public void distributeMoneyIfExact(ArrayList<String> borrowers,String lender,ArrayList<Integer> exactShare)

```

Comment:
instead of exposing multiple methods from here. You could have created a single method like addExpense and taken an expense object as param. And deciding which distribution to choose could be done here